### PR TITLE
added singular amenity component

### PR DESCRIPTION
**Pull request by jlavendtt**

### CODE REVIEW

File: src/components/Amenities/Amenities.vue [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
       }
   },
+  components: {
+    Amenity
+
+  },
 
   mounted(){
-        axios.get('http://localhost:8080/api/amenity/getByListingId/1')
+
```
Comment (Eliotdoesprogramming) [inline src/components/Amenities/Amenities.vue:34]: pass down listing as a prop

File: src/components/Amenities/Amenities.vue [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+      let listingId = this.$route.params.id;
+        axios.get('http://localhost:8080/api/amenity/getByListingId/'+listingId)
```
Comment (Eliotdoesprogramming) [inline src/components/Amenities/Amenities.vue:36]: Listing is already available on the parent component, pass down as a prop

**Review by Eliotdoesprogramming — CHANGES_REQUESTED** [review]

**Review by jaitken — APPROVED** [review]
Looks good!

**Review by Eliotdoesprogramming — APPROVED** [review]
lgtm

**Pull request merged**